channels/email/accounts/account-setup.rst

```diff
@@ -19,11 +19,15 @@ Setting up a new email account? Here’s what all the settings do.
    and to turn it back on once all your messages have been imported.
 
 .. note:: **Gmail / G Suite users:**
-   Google is in the process of upgrading its security policies.
-   To stay up-to-date, do **not** add your account as an email channel—instead,
-   create a :doc:`Google channel </channels/google/accounts/account-setup>`.
-
-   **Office365 / Exchange users:** Shared mailboxes are not supported.
+      Google is in the process of upgrading its security policies.
+      To stay up-to-date, do **not** add your account as an email channel—instead,
+      create a :doc:`Google channel </channels/google/accounts/account-setup>`.
+   **Microsoft 365 users:**
+      Microsoft is in the process of upgrading its security policies.
+      To stay up-to-date, do **not** add your account as an email channel—instead,
+      create a :doc:`Microsoft 365 channel </channels/microsoft365/accounts/account-setup>`.
+
+   **Microsoft 365 / Exchange users:** Shared mailboxes are not supported.
```

Comment:
I really like this idea of using RST definition lists in larger, more complicated notes like this one.

For better legibility/formatting, I'd prefer something like this:

```rst
.. note::

   **Gmail / G Suite users:**
      Google is in...

   **Microsoft 365 users:**
      Microsoft is in...

   **Microsoft 365 / Exchange users:**
      Shared mailboxes are not supported.
```

but GitHub won't let me add this as a code suggestion, so I'll just push it in a new commit.